tools/test/topology/test-multi-playback.m4

```diff
@@ -0,0 +1,114 @@
+#
+# Topology for pass through pipeline
+#
+
+# Include topology builder
+include(`pipeline.m4')
+include(`dai.m4')
+include(`ssp.m4')
+include(`utils.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
+
+DEBUG_START
+
+#
+# Machine Specific Config - !! MUST BE SET TO MATCH TEST MACHINE DRIVER !!
+#
+# TEST_PIPE_NAME - Pipe name
+# TEST_DAI_LINK_NAME - BE DAI link name e.g. "NoCodec"
+# TEST_DAI_PORT	- SSP port number e.g. 2
+# TEST_DAI_FORMAT - SSP data format e.g s16le
+# TEST_PIPE_FORMAT - Pipeline format e.g. s16le
+# TEST_SSP_MCLK - SSP MCLK in Hz
+# TEST_SSP_BCLK - SSP BCLK in Hz
+# TEST_SSP_PHY_BITS - SSP physical slot size
+# TEST_SSP_DATA_BITS - SSP data slot size
+# TEST_SSP_MODE - SSP mode e.g. I2S, LEFT_J, DSP_A and DSP_B
+#
+
+# Apply a non-trivial filter blob IIR and FIR tests. TODO: Note that the
+# PIPELINE_FILTERx notation will be updated in future for better flexibility.
+define(PIPELINE_FILTER1, ifelse(TEST_PIPE_NAME, `eq-iir', `eq_iir_coef_loudness.m4'))
+define(PIPELINE_FILTER2, ifelse(TEST_PIPE_NAME, `eq-fir', `eq_fir_coef_loudness.m4'))
+
+#
+# Define the pipeline
+#
+# PCM0P  -->  TEST_PIPE_NAME  -->  SSP(TEST_DAI_PORT)
+#
+
+# Playback pipeline 1 on PCM 0 using max 2 channels of s32le.
+# Set 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-TEST_PIPE_NAME-playback.m4,
+	1, 0, 2, s32le,
+	1000, 0, 0,
+	8000, 192000, 48000)
+
+# Playback pipeline 2 on PCM 1 using max 2 channels of s32le.
+# Set 1000us deadline on core 0 with priority 1
+PIPELINE_PCM_ADD(sof/pipe-passthrough-playback.m4,
+	2, 1, 2, s32le,
+	1000, 1, 0,
+	8000, 192000, 48000)
+
+# DAI configuration
+#
+# SSP port TEST_DAI_PORT is our only pipeline DAI
+#
+
+# playback DAI is SSP TEST_DAI_PORT using 2 periods
+# Buffers use s24le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-playback.m4,
+	1, TEST_DAI_TYPE, TEST_DAI_PORT, TEST_DAI_LINK_NAME,
+	PIPELINE_SOURCE_1, 2, TEST_DAI_FORMAT,
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# playback DAI is SSP TEST_DAI_PORT using 2 periods
+# Buffers use s24le format, with 48 frame per 1000us on core 0 with priority 1
+DAI_ADD(sof/pipe-dai-playback.m4,
+	2, TEST_DAI_TYPE, TEST_DAI_PORT, TEST_DAI_LINK_NAME,
+	PIPELINE_SOURCE_2, 2, TEST_DAI_FORMAT,
+	1000, 1, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# connect pipelines together
+SectionGraph."pipe-sof-second-pipe" {
+        index "0"
+
+        lines [
+		# keyword detect
+                dapm(PIPELINE_CROSSOVER_1, PIPELINE_PCM_2)
```

Comment:
I think this might be backwards it should be (sink, source)
see https://github.com/thesofproject/sof/blob/master/tools/topology/sof-byt-codec.m4